### ISSUE TITLE
Cube is binding itself to every zigbee device in my house

### ISSUE THREAD

**Issue by amaisano** [JJPro/CTP-R01-converter#3]
When this paired with Z2M, any action the cube performs is toggling on and off all my lights, switches, EVERYTHING.

What on earth? I've disabled it from HA, but even then it's still interacting with lights across the house. There are no data bindings setup in Z2M.

When I rotate the cube, i get the below. It seems to be sending onOff commands to EVERYTHING...

```
2023-01-23 17:06:02Received Zigbee message from 'Owl', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from '0x54ef4410006a4928', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 1 with groupID 0
2023-01-23 17:06:02No converter available for 'CTP-R01' with cluster 'genOnOff' and type 'commandOff' and data '{}'
2023-01-23 17:06:02Received Zigbee message from 'Washing Machine Outlet', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
```

**Comment by JJPro** [comment]
That's quite odd, a few others reported same issue long ago, then I added the following to ignore `genOnOff` cluster: 

```js
  fromZigbee: [aqara_opple, action_multistate, action_analog, fz.ignore_onoff_report],
```

In fact, I never saw `genOnOff` cluster messages all this time, even with the sniffer. 

Can you verify you are with the latest version of Z2M and HA?

**Comment by JJPro** [comment]
I wonder whether there exists different versions of this cube, or you cube needs an firmware update. 
I got my cube from Amazon on Dec 18th, when and where do you get yours? 

I'm with the latest firmware 0.0.0_0025, what about yours? (on device's *about* tab)

**Comment by JJPro** [comment]
Moreover, have you tried restart z2m and HA after putting the code in place?

**Comment by amaisan0** [comment]
Got mine the first week of January from Amazon US.

I had it pair with an Aqara G3 hub until today.

Firmware build date
20220602
Firmware version
0.0.0_0025

I'm on the latest stable HA and Z2M addon.

I'm using a zzh! stick as my coordinator.

**Comment by JJPro** [comment]
Another thing, this converter doesn't bind the device to anything. 

<img width="920" alt="image" src="https://user-images.githubusercontent.com/871037/214171473-674ede1f-1e4b-456b-a881-9e09dbb10d1e.png">

See? My bind list is empty. 

I saw someone else's [code](https://github.com/Koenkk/zigbee2mqtt/issues/15652) binds `genOnOff` with coordinator. 

Can you make sure my converter, and only my converter for this particular device, is actually in effect? 

To test my code is in effect, you can put `console.log('>>> hi there')` in one of the `converter` callbacks, e.g. in `action_multistate`'s `converter` callback: 
```js 
const action_multistate = {
  cluster: 'genMultistateInput',
  type: ['attributeReport', 'readResponse'],
  options: [],
  convert: (model, msg, publish, options, meta) => {
    console.log('>>> hi there');
    const value = msg.data['presentValue'];
    let payload;
    ...
  }
};
```

Then see if you can find ">>> hi there" in your logs. 

Also, check in Settings > External Converters see if you find other converters there: 
<img width="887" alt="image" src="https://user-images.githubusercontent.com/871037/214173662-4a0b1ee6-48d7-4b50-be2b-874c8ee62d04.png">

**Comment by JJPro** [comment]
FYI, I updated my last message, refresh.

**Comment by JJPro** [comment]
oh, Z2M needs to be restarted whenever code is changed.

**Comment by amaisan0** [comment]
It's the only external converter I have, copied 1:1 from your repo. I restarted the Z2M addon after adding it. No binds for me either, but that was what I thought was happening because I had no automations that did such things.

**Comment by JJPro** [comment]
Interesting, I've had the sniffer running for like 30mins now after seeing your issue report, and have been rotating, flipping.... No packets from Cluster `genOnOff` have been caught. 

Just placed another order on Amazon, see if I can get this issue on the new one. Wait for two days.

**Comment by amaisan0** [comment]
I force removed the cube and set it to ONLY join to the main coordinator (something I've had to do with many other Aqara Xiaomi devices). 

I kept pressing the link button on the cube and performing the shake action the first time to ensure communication during pairing, so THIS time I didn't touch anything after initiating the pair.

And now it seems to be working without affecting the rest of my Zigbee devices.

It's in scene mode right now so I will also test in action mode.
